### ISSUE TITLE
Add SS Laises Faire to Mission Files

### ISSUE THREAD

**Issue by bleehu** [bleehu/CXDocs#182]
Add layout, puzzle, enemy stats and loot to the mission files tool!

**Comment by bleehu** [comment]
Blocked by merge of PR #180

**Comment by bleehu** [comment]
Blocked by work needed to can SSLaissezFaire

**Comment by bleehu** [comment]
Won't Do
